support/index.markdown

```diff
@@ -12,11 +12,11 @@ title: Support
 
   * First, check out the list of [Frequently Asked Questions](/documentation/faqs) to see if your question has been answered already.
```

Comment:
I actually found the FAQ can be still helpful particularly for new users of MoveIt! and/or ROS. I found it addresses some of the things that new people can get confused. So I +1 for keeping it at the top of the list.
In fact I added another commit to this PR for minor changes to FAQ page.